Sources/SwiftSyntaxBuilder/StringConvenienceInitializers.swift

```diff
@@ -59,3 +59,13 @@ extension String {
     IdentifierPattern(self)
   }
 }
+
+/// `String` conforms to `SyntaxBuildable` and `ExprBuildable` via different paths, so we need to pick one default conversion path to create an an `ExprSyntax` (and `Syntax`) from a `String`. We choose `IdentifierExpr`.
+extension String {
+  public func createExprBuildable() -> ExprBuildable {
+    return createIdentifierExpr()
+  }
+  public func createSyntaxBuildable() -> SyntaxBuildable {
+    return createIdentifierExpr()
```

Comment:
Good idea 👍.

I added the `return`s again because I think that the gyb files are hard enough to read as-is, so I wanted to make as much behavior explicit as possible and I think it removes overall readability. 

Also I’m personally not a huge fan of implicit returns inside functions, it just doesn’t feel right to me (but I like it in closures).

Reply:
It makes total sense to use it in gyb!